handbook/product/README.md

```diff
@@ -34,6 +34,9 @@ At Fleet, like [GitLab](https://about.gitlab.com/handbook/product-development-fl
 - Prepare your draft in the user story issue.
 - Prepare the agenda for your design review meeting, which should be an empty document other than the proposed changes you will present.
 - Review the draft with the CEO at one of the daily design review meetings, or schedule an ad-hoc design review if you need to move faster.  (Efficient access to design reviews on-demand [is a priority for Fleet's CEO](https://fleetdm.com/handbook/business-operations/ceo-handbook). Emphasizing design helps us live our [empathy](https://fleetdm.com/handbook/company#empathy) value.)
+- When introducing a story, clarify which review "mode" the CEO should operate in:
+  + **Final review** mode — you are 70% sure the design is 100% done. If you've engaged your PM or engineering in drafting the story, you've aligned on a solution.
```

Comment:
"...and you are aligned on the solution with engineering."

Reply:
@noahtalerman What about just leaving it at "you are 70% sure the design is 100% done"?
My reasoning: We already say to engage engineering in the section above this pretty prominently. Including it as a hard requirement here doesn't entirely reflect my experience designing for CX so far— the level of engineering input has varied a lot. I think if I were new to Fleet reading this, I would assume that I should never bring something to Mike for final review without showing to engineering first (which perhaps is the intent here anyway?)

Happy to make your suggested change btw— just wanted to voice my thoughts first since whatever we write down here, I intend to follow, and it'll be a slight change to my process to engage engineering for _every_ design (though totally doable if that's what's best!)

Reply:
@rachaelshaw leaving it at "you are 70% sure the design is 100% done" makes sense to me!

I agree with your reasoning. Right now "...and you are aligned on the solution with engineering." makes it sound like this is a requirement when I agree that it shouldn't be.

And, as you noted, we call out engaging engineering in the section above.

Reply:
👍 thanks! Made that change.